resources/public/api/conf/1.0/examples/LISAAccount.close.response.json

```diff
@@ -1,6 +1,6 @@
 {
   "data": {
-    "accountId": "8765432100",
+    "accountId": "9876543210",
```

Comment:
Would be worth using the same accountId here as what we use in the first successful test scenario (1234567890)